app/demo-loop/Transporter.cc

```diff
@@ -294,7 +278,6 @@ TransporterResult Transporter<M>::operator()(const TrackInitParams& primaries)
         {
             actions.invoke<M>(action, core_ref);
         }
-        accum_time<M>(input_, get_time, &result.time.launch_models);
```

Comment:
I think we can keep the timer :). We weren't actually calling `launch_models` anymore.